commandapi-platforms/commandapi-velocity/commandapi-velocity-plugin/src/main/java/dev/jorel/commandapi/config/VelocityConfigurationAdapter.java

```diff
@@ -102,9 +87,9 @@ public void saveDefaultConfig(File directory, File configFile, Logger logger) {
 				logger.severe("Failed to create directory for the CommandAPI's config.yml file!");
 			}
 			try {
-				ConfigurationAdapter<ConfigurationNode> velocityConfigurationAdapter = new VelocityConfigurationAdapter(configLoader, configLoader.createNode(), defaultConfig);
-				configGenerator.populateDefaultConfig(velocityConfigurationAdapter);
-				configLoader.save(velocityConfigurationAdapter.config());
+				ConfigurationAdapter<ConfigurationNode> velocityConfigurationAdapter = new VelocityConfigurationAdapter(loader, loader.createNode(), defaultConfig);
+				configGenerator.generate(velocityConfigurationAdapter);
+				loader.save(velocityConfigurationAdapter.config());
```

Comment:
> Although, I still think merging this logic between Bukkit and Velocity in `ConfigGenerator` would help avoid issues like this where the code that should do the same thing is correct in one version but has a bug in the other.

I know you want me to put the `saveDefaultConfig()` method into the `ConfigGenerator` class, however as mentioned already, I *really* don't think that that method is correct in there and I **really** don't want to put it in there. Although, I have another idea to avoid issues like this.